project-koku/koku
# Dropping filtering of deleted OUs from query

Fixes [COST-2631](https://issues.redhat.com/browse/COST-2631)

Changes proposed in this PR:
* Dropping filter of deleted OUs from resource type endpoint so return matches standard OU endpoint.

Testing Instructions:
1. Add AWS source
2. Populate some org units with deleted values
3. Should see both resource-types/aws-organizational-units/ and organizations/aws/ match in return count.


NOTES:
Reach out for help with creating the org tree if needed
